docs/releases/1.10.txt

```diff
@@ -26,6 +26,17 @@ What's new in Django 1.10
 
 ...
 
+Bug fixes
```

Comment:
You can remove this unless there are any backwards-incompatible possibilities which should be documented in the existing section of this document.